tests/utils.go

```diff
@@ -239,17 +268,23 @@ func AfterTestSuitCleanup() {
 	createNamespaces()
 	cleanNamespaces()
 
-	deletePVC("alpine", false)
-	deletePV("alpine", false)
+	deletePVC(osCirrosNFS, false)
+	deletePV(osCirrosNFS, false)
```

Comment:
Are we testing the nfs demo PVC at all?  I didn't see either of these in a storage test.

Reply:
@davidvossel You are right, I just want to be sure that we want to leave NFS server, if yes, I will add additional tests to check PVC with NFS backend.